static/scripts/dashboard/dashboardCtrl.js

```diff
@@ -2288,13 +2144,7 @@
     function updateSystemSettings(next) {
       if (!$scope.vm.systemSettingsId) return next();
 
-      var update = {};
-
-      _.each($scope.vm.installForm.systemSettings,
-        function (setting) {
-          update[setting.name] = setting.value;
-        }
-      );
+      var update = $scope.vm.installForm.systemSettings;
```

Comment:
This could reset the version.  We should only update fields that can safely change from the UI.